DigitalLearningSolutions.Web/Views/TrackingSystem/Centre/Reports/_FilterDisplayCard.cshtml

```diff
@@ -33,7 +33,7 @@
             Date range
           </dt>
           <dd class="nhsuk-summary-list__value">
-            @Model.DateRange
+            <span>@Model.StartDate</span><span class="screen-reader-only">to</span><span> - @Model.EndDate</span>

```

Comment:
Is there a reason for this screen-reader-only class? rather than using the nhsuk-u-visually-hidden class?